product_docs/docs/pgd/5/quickstart/further_explore_replication.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: Exploring replication with PGD
+navTitle: Exploring replication
+deepToC: true
+---
+
+## Explore replication with PGD
+
+With the cluster up and running, it's worthwhile to run some basic checks to see how effectively it's replicating.
+
+The following example shows one quick way to do this, but you must ensure that any testing you perform is appropriate for your use case.
+
+* **Preparation**
+  * Ensure the cluster is ready:  
```

Comment:
Perhaps this can be a link to https://www.enterprisedb.com/docs/pgd/latest/quickstart/?

Reply:
Thats a tad recursive.... this is ready as in quiescent, not ready as in installed.

Reply:
Also the instructions on how to do that are explicitly after that srep.

Reply:
True it's just for the scenario where I send someone a direct link to this page.

Reply:
But also, you'd probably want the version in manual install (once it is synced back up with this one) as that deals in hosts and nodes in the abstract, not the named instances as per the tpa install.

Reply:
It's up to you

Reply:
```suggestion
  * Ensure your cluster is ready to perform replication. If you have not installed a cluster yet, use one of the [quickstart](.) guides to get going:  
```